### ISSUE TITLE
How much is the per gpu batch size?

### ISSUE THREAD

**Issue by qiao1025566574** [zdou0830/METER#18]
How much is the per gpu batch size? 
total batchsize is 4096, GPU num is 8, so per gpu batch size is 512?
But I use A100 GPU, the batch size only can be set 16?

**Comment by zdou0830** [comment]
You can set the per_gpu_batch_size based on your GPU memory. The gradients will accumulate so it doesn't affect the total batch size.

**Comment by qiao1025566574** [comment]
Thank you!

**Comment by qiao1025566574** [comment]
Another question: How to computer the epoch? Like this?
Epoch = Batchsize * step / len(dataset) = 4096 * 100000 / 4000000 = 102?

**Comment by zdou0830** [comment]
For our pre-training datasets, the number of the total image-caption pairs is about 10M, so the number of training epochs is about 4096*10k/10M $\approx$ 40.

**Comment by qiao1025566574** [comment]
OK! Thank you for your reply. @zdou0830
